tests/streamingcli/jupyter/test_notebook_converter.py

```diff
@@ -327,6 +330,30 @@ def filter_print(condition, message):
     return condition
 
 
-t_env.create_temporary_function("filter_print", filter_print)
+table_env.create_temporary_function("filter_print", filter_print)
 '''
         )
+
+    def test_notebook_with_flink_code(self):
+        # given
+        file_path = "tests/streamingcli/resources/jupyter/notebook7.ipynb"
+        # expect
+        converted_notebook = convert_notebook(file_path)
+        assert (
+            converted_notebook.content
+            == """from pyflink.common.typeinfo import Types
+from pyflink.datastream import StreamExecutionEnvironment
+from pyflink.table import StreamTableEnvironment, DataTypes
```

Comment:
I see that DataTypes and udf are used in one of the tests and they are imported in the flink.app.template.
What about removing those imports from all tests that do not refer to udf and DataTypes and adding them only where we use udf and DataTypes? Would that be a good idea? Besides that LGTM!

Reply:
Hmmm, I think we don't need these imports in the template at all. We should have them in the code.

Reply:
User who defines function via jupyter should add imports as those are part of function definition. Imo no need to have them in template, especially in scenario when only sql is submitted

Reply:
I removed them in the last commit